mailer/README.md

```diff
@@ -0,0 +1,105 @@
+# github.com/gobuffalo/x/mailer
+
+This package is intended to allow easy Email sending with Buffalo, it allows you to define your custom `mailer.Sender` for the provider you would like to use.
+
+The following is an example on how to setup a smtp mailer as well as creating a Mailer function to use the 
+
+```go
+//actions/mailer.go
+
+import (
+    "github.com/gobuffalo/x/mailer"
+    "github.com/gobuffalo/packr"
+    "github.com/me/myapp/models"
+    "github.com/pkg/errors"
+)
+
+var smtp mailer.Sender
+
+func init() {
+	port := envy.Get("SMTP_PORT", "1025")
+	host := envy.Get("SMTP_HOST", "localhost")
+	user := envy.Get("SMTP_USER", "")
+	password := envy.Get("SMTP_PASSWORD", "")
+
+	var err error
+	smtp, err = mailer.NewSMTPSender(host, port, user, password)
+
+	if err != nil {
+		log.Fatal(err)
+	}
+}
+
+//SendContactMessage Sends contact message to contact@myapp.com
+func SendContactMessage(c *models.Contact) error {
+    
+	//Creates a new message
+	m := mailer.NewMessage()
+	m.From = "sender@myapp.com"
+	m.Subject = "New Contact"
+	m.To = []string{"contact@myapp.com"}
+
+	// Data that will be used inside the templates when rendering.
+	data := map[string]interface{}{
+		"contact": c,
+	}
+
+	// You can add multiple bodies to the message you're creating to have content-types alternatives.
+	err := m.AddBodies(data, r.HTML("mail/contact.html"), r.Plain("mail/contact.txt"))
```

Comment:
Where does `r` come from in this example? We should show setting it up so people don't just cut and paste and can't figure out what it comes from.